post-processor/digitalocean-import/post-processor.go

```diff
@@ -119,7 +119,6 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		"spaces_region": &p.config.SpacesRegion,
 		"space_name":    &p.config.SpaceName,
 		"image_name":    &p.config.Name,
-		"image_regions": &p.config.ImageRegions[0],
```

Comment:
I think this "image_regions" should be set again later on some where ? Or is it not needed at all ?

Reply:
This map is just used to loop through the required arguments and check that they have been configured. When used latter, it is accessed from the config object: 

https://github.com/hashicorp/packer/blob/ae1bb856a31d0e0846b4dce3db38850831470e90/post-processor/digitalocean-import/post-processor.go#L276

Reply:
Oh, yes I understand it better now, LGTM 🙂. Thanks for explaining.

Reply:
Thanks for the review!